api-reference/10 UI Components/dxCheckBox/1 Configuration/value.md

```diff
@@ -8,18 +8,18 @@ default: false
 Specifies the UI component state.
 
 ---
-The check box can be in one of the following states depending on the **value** property value.
+The check box can be in one of the following states depending on the **value** property's value.
 
 - checked  
- value is **true**
+ **value** is **true**
 
 - unchecked  
- value is **false**
+ **value** is **false**
 
 - undetermined  
- value is **undefined**
+ **value** is **undefined** or **null**
 
-An end-user can change the UI component value using the **Space** key.
+Users can press the Space key to change the **value**.
```

Comment:
Can we add bold font to "Space" as well? For example:

`Users can press the **Space** key to change the **value**.`